packages/components/src/InputNumber/InputNumber.tsx

```diff
@@ -18,6 +18,11 @@ interface InputNumberProps
       | "defaultValue"
     > {
   value?: number;
+
+  /**
+   * Allow keyboards for mobile to display decimal option
+   */
+  allowDecimals?: boolean;
```

Comment:
Would allowing this component to `Pick` the keyboard prop with a default of `numeric` have the same effect?

Reply:
Actually, I think that may would be better. Let me update things